src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/CoreCLR/InvokeWebRequestCommand.CoreClr.cs

```diff
@@ -63,7 +63,7 @@ internal override void ProcessResponse(HttpResponseMessage response)
             {
                 string outFilePath = WebResponseHelper.GetOutFilePath(response, _qualifiedOutFile);
 
-                WriteVerbose(string.Create(System.Globalization.CultureInfo.InvariantCulture, $"File Name: {Path.GetFileName(_qualifiedOutFile)}"));
+                WriteVerbose(string.Create(System.Globalization.CultureInfo.InvariantCulture, $"File Name: {response.RequestMessage?.RequestUri?.Segments[^1]}"));
```

Comment:
Wouldn't we want to use the `outFilePath` rather than repeating the more complex logic done in `GetOutFilePath`? We can keep the `Path.GetFileName` call if we just want the filename and not the full path.

Reply:
@jborean93 , good catch! That is a much better way to do it.